src/deluge/model/global_effectable/global_effectable.cpp

```diff
@@ -156,11 +168,23 @@ void GlobalEffectable::displayCompressorAndReverbSettings(bool on) {
 	}
 	else {
 		if (on) {
-			display->displayPopup(getCompressorModeDisplayName());
+			if (getSelectedClip()) {
+				// We are inside a clip (either audioclip or affect-entire kit): show Sidechain sync
```

Comment:
The comment should just say "We are inside a clip : show Sidechain sync" - the master compressor is not in any of the clips, so it should be side chain everywhere outside song.